plugins/grafana-sankey-plugin/src/SankeyPanel.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { PanelProps } from '@grafana/data';
+import { SankeyOptions } from 'types';
+import Chart from 'react-google-charts';
+
+interface Props extends PanelProps<SankeyOptions> {}
+
+export const SankeyPanel: React.FC<Props> = ({ options, data, width, height }) => {
+  let result = [
+    ['From', 'To', 'Bytes'],
+    ['Source N/A', 'Destination N/A', 1],
```

Comment:
is the 2nd line above a default text to display when there's no data?

In that case can we set bytes to 0?

Reply:
yes. They do not allow 0 as value, that's why I use 1 instead.

Reply:
Got it, thanks!